src/person.rb

```diff
@@ -1,14 +1,21 @@
-class Person
+require('./src/nameable')
```

Comment:
* Kindly fix the relative parts to your files as i get an error i noticed your path is written this way `./src/nameable` Try writting it this way instead `./nameable` do same for all your classes 
![image](https://github.com/Abdelaziz-Mahdi/school-library/assets/52497246/d6e20adb-2099-454f-b091-6d8d56d0deff)

Reply:
_Hello @Davidosky007_ 

Thank you for reviewing my code, in your ss I saw that you have 
navigated to **` school-library\src`** and used **`ruby person.rb `**
I would suggest the following, 
navigate to **`school-library`**  and run **`ruby src/person.rb`**
Kindly let me know if that is not possible, and I`ll change my files.

_I appreciate your time,
Best wishes._